ts/configuration/v1.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Exposes the Kubernetes extension's Configuration settings
+ */
+export interface ConfigurationV1 {
+    /**
+     * Gets the path for the kubeconfig currently in use.
+     * @returns a KubeconfigPath designating the environment the path relates to, and the path.
+     */
+    getKubeconfigPath(): ConfigurationV1.KubeconfigPath;
+}
+
+export namespace ConfigurationV1 {
+    /**
+     * Represents the path to the kubeconfig on the host machine.
+     */
+    export interface HostKubeconfigPath {
+        /**
+         * The environment the path relates to
```

Comment:
Nitpick: doc strings should end with a full stop (.), even for sentence fragments.